features/homepage.feature

```diff
@@ -9,14 +9,13 @@ Feature: Homepage
     And there is a public configuration created named "Test Configuration"
     And I have a public repository named "Test Repository"
     Then I am at the homepage
-    And I should see "Home"
     And I should see "Projects"
     And I should see "Repositories"
     And I should see "Configurations"
     And I should see "Reading Groups"
     And I should see "Sign In"
     And I should see "Sign Up"
-    And I should see "Language"
+    And I should see the "en" flag
```

Comment:
Our team discussed and we think it's better to keep this implementation.

In the step definition we already are using `I18n.locale` (as you can see [here](https://github.com/mezuro/prezento/pull/392/files#diff-0d10c2312bb5ec92959713d01cfc9fa9)) and comparing it to the locale that we receive from the scenario.

Another thing that we can keep in mind is that this scenario is not changeable, which means that this scenario is always going to be in english, therefore, making unnecessary even passing the `en` locale as parameter.